data-raw/lookup_msoa_lad_19.R

```diff
@@ -0,0 +1,32 @@
+# ---- Load ----
+library(tidyverse)
+library(devtools)
+library(sf)
+
+# Load package
+load_all(".")
+
+# Set query url
+query_url <-
+  query_urls |>
+  filter(id == "msoa_lad_19") |>
+  pull(query)
+
+msoa_lad <-
+  read_sf(query_url)
+
+# Select and rename vars
+msoa_lad <-
+  msoa_lad |>
+  select(
+    msoa_11_name = MSOA11NM,
```

Comment:
I think the fact that some lookup tables use different year codes creates a problem. It isn't clear from just the name of the variable alone, what year the lookups cover. For example for this variable `lookup_msoa_lad_19`, I would expect both the msoa and lad codes to be 2019.

I think a better naming system would be to use `lookup_geographr1_year1_geography2_year2`. In this instance that would mean renaming the file, variable, query-urls.R table, and data.R documentation to use lookup_msoa_11_lad_19.

Do you agree with this new system? If so please can you update the lookups to match this new schema? The lookup `lookup_lad_lad` can be left as is, as it covers many years.

Reply:
Yes good idea, particularly with the new census data coming. Have made the changing to names in commit https://github.com/humaniverse/geographr/pull/24/commits/a6f1fca2b0e9106ad83d0ac04998ab554e1d289f.

Reply:
Also I will update demograhr so it has the same naming convention for column names making the year codes explicit e.g. `lad_21_name` vs. `lad_name` to make joining between tables in geographr and demographr.

Reply:
Thanks for the updates